### DISCUSSION TITLE
3/24(금) 복습

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#30] [General]
오늘은 요번주에 한 내용을 복습하는 것입니다. 하지만 제가 알기로는 정처기가 얼마 남지않았다고 알고 있습니다. 오늘은 간단히 정리하고 정처기 하는 것을 추천 드립니다. 정처기 열심히 하시고 제가 토요일에는 그냥 좀 재밌는 숙제 내겠습니다. 가능한이면 맥북왔다고 맥북만 만지지말고 정처기는 가능하면 1학년 때 따야합니다. 열심히하세요~ 🫠

**Comment by Eunho0922** [comment]
넹 꼭 한번에 합격해보겠습니당!
